Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Android/Setup.cs

```diff
@@ -19,19 +27,29 @@ protected override IMvxIoCProvider CreateIocProvider()
         {
             var containerBuilder = new ContainerBuilder();
             containerBuilder.RegisterModule<ApplicationModule>();
+            containerBuilder.Register(c => Environment.GetFolderPath(Environment.SpecialFolder.Personal)).Named<string>(App.BootstrapParamaters.LocalAppDataFolder);
```

Comment:
Why using the value from `App.BootstrapParamaters.LocalAppDataFolder` (which could be, `/home/someUser/AppData`) instead of using just the value `"localAppData"`. This is a registration, so the actual name of the registration does not need to be the actual contents. (In fact, I think it shouldn't).

So, right now you'll have in your registrations table:

Registration Name: `/home/someUser/AppData`, Value: `/home/someUser/Personal`

I suggest we should have:

Registration Name: `ApplicationData`, Value: `/home/someUser/Personal`